### ISSUE TITLE
Help with new virtual environment

### ISSUE THREAD

**Issue by skinnedknees** [prof79/fansly-downloader-ng#95]
Hi. I was using the Windows executable here and there, but it stopped working for me. Someone suggested trying the python version and referred to a write-up that a user put together. Well I followed it and it worked great. My only problem is my PC rebooted before I copied down the commands to execute it, and now I dont know how to get "in" to the virtual python environment(I think the prompt changed to venv>?)

This was the thread and comment here > https://github.com/prof79/fansly-downloader-ng/issues/80#issuecomment-2249261123

But for some reason instead of that thread getting closed it looks like it was deleted entirely.

If anyone knows what those directions were, can you can post how to start the virtual environment for me. I cant find it anywhere else

Thanks

**Comment by wrigglyb** [comment]
Assuming you called the venv venv.. On Windows if you are using Powershell or Terminal then you are looking to run `Activate.ps1` located inside the Scripts folder inside the folder you made the venv in. If you are on CMD then it's all lowercase and it's .bat. Capitalisation definitely matters.

The most straightforward way to do it is to go back to the place you made it and run 

`.\venv\Scripts\activate.bat` or `.\venv\Scripts\Activate.ps1`

from the point where you made the venv, like so:

<img width="589" alt="Screenshot 2024-10-12 at 10 32 17" src="https://github.com/user-attachments/assets/bb180911-3a28-472f-bccc-f03e7b3424e7">

You will get the venv in the prompt back and be in your venv. To exit it voluntarily just send `deactivate`.

**Comment by skinnedknees** [comment]
I'm not sure what the virtual environment was called, is there a path or command that lists them? I just followed the guide that was posted. I did run your commands above(you can see I got the (venv) prompt), but it didn't like something about them. See below...

(venv) PS Z:\Fansly> .\venv\Scripts\Activate.ps1
(venv) PS Z:\Fansly> python -m fansly_downloader_ng.py
Traceback (most recent call last):
  File "<frozen runpy>", line 189, in _run_module_as_main
  File "<frozen runpy>", line 112, in _get_module_details
  File "Z:\Fansly\fansly_downloader_ng.py", line 52, in <module>
    from updater import self_update
  File "Z:\Fansly\updater\__init__.py", line 8, in <module>
    from .utils import check_for_update, delete_deprecated_files, post_update_steps
  File "Z:\Fansly\updater\utils.py", line 15, in <module>
    from pkg_resources._vendor.packaging.version import parse as parse_version
ModuleNotFoundError: No module named 'pkg_resources._vendor'

Does that mean anything to you? Is there a way I can just start it over like I did from that guide?

**Comment by palmerbomber1** [comment]
I think I replied to you in another issue? Just in case tho, Ill share the link to it from web archive [here](http://web.archive.org/web/20241001085407/https://github.com/prof79/fansly-downloader-ng/issues/80)

**Comment by skinnedknees** [comment]
Omg thank you! That's it! I think I screwed things up while trying to get it working(without the guide). It was giving me a "Numpy" error. So I just ran

> python -m venv .venv
> .\.venv\Scripts\activate
> python -m pip install -r requirements.txt --force (the 'force' was key I think)
> python fansly_downloader_ng.py

And its running great again. Thanks for the replies and getting me back on track. Hopefully that web archive [link](http://web.archive.org/web/20241001085407/https://github.com/prof79/fansly-downloader-ng/issues/80) will help other people switching from a windows binary to python environment
